README.md

```diff
@@ -49,41 +49,56 @@ $ sha256sum -c --strict SHA256SUM.md
 
 ## Verification of Provenance
 
+We currently support artifact verification (for binary blobs) and container images.
+
 ### Available options
 
-Below is a list of options currently supported. Note that signature verification is handled seamlessly without the need for developers to manipulate public keys.
+Below is a list of options currently supported for binary blobs and container images. Note that signature verification is handled seamlessly without the need for developers to manipulate public keys. See [Available options](#available-options) for details on the options exposed to validate the provenance.
 
 ```bash
 $ git clone git@github.com:slsa-framework/slsa-verifier.git
-$ go run ./cli/slsa-verifier --help
- Usage of ./slsa-verifier:
-  -artifact-path string
-    	path to an artifact to verify
-  -branch string
-    	expected branch the binary was compiled from (default "main")
-  -print-provenance
-    	output the verified provenance
-  -provenance string
-    	path to a provenance file
-  -source string
-    	expected source repository that should have produced the binary, e.g. github.com/some/repo
-  -tag string
-    	[optional] expected tag the binary was compiled from
-  -versioned-tag string
-    	[optional] expected version the binary was compiled from. Uses semantic version to match the tag
+$ go run ./cli/slsa-verifier/ verify-artifact --help
+Verifies SLSA provenance on an artifact blob
+
+Usage:
+  slsa-verifier verify-artifact [flags]
+
+Flags:
+      --branch string            [optional] expected branch the binary was compiled from
+      --builder-id string        EXPERIMENTAL: the unique builder ID who created the provenance
+  -h, --help                     help for verify-artifact
+      --print-provenance         print the verified provenance to std out
+      --provenance-path string   path to a provenance file
+      --source string            expected source repository that should have produced the binary, e.g. github.com/some/repo
```

Comment:
do we need to rename to `-source-uri`? How about `-source-tag`, `-source-branch` and `source-versioned-tag`?
I am trying to think whether we may hit name collision in the future, like `-container-tag` or something else.

Also, since SLSA has 3 main requirements: `source`, `build` and `provenance`, I'm wondering if we should try to align with this as well.

Reply:
I agree that `source` should prefix the `tag` and `branch` and URI should be qualified for what `source` means. Added for `source`